001__initial_db_setup.py

```diff
@@ -0,0 +1,62 @@
+import sqlite3
+import argparse
+
+
+def create_tables(unique_by_fields=False):
+    conn = sqlite3.connect('database.db')
+    db_cursor = conn.cursor()
+
+    db_cursor.execute('''
+                    CREATE TABLE IF NOT EXISTS Bank (
+                        id INTEGER PRIMARY KEY,
+                        name TEXT NOT NULL UNIQUE
+                    )
+                ''')
+
+    db_cursor.execute('''
+                            CREATE TABLE IF NOT EXISTS Transact (
+                                id INTEGER PRIMARY KEY,
+                                Bank_sender_name TEXT NOT NULL,
+                                Account_sender_id INTEGER NOT NULL,
+                                Bank_receiver_name TEXT NOT NULL,
+                                Account_receiver_id INTEGER NOT NULL,
+                                Sent_Currency TEXT NOT NULL,
+                                Sent_Amount REAL NOT NULL,
+                                Datetime TEXT
+                            )
+                        ''')
+
+    db_cursor.execute('''
+                        CREATE TABLE IF NOT EXISTS User (
+                            id INTEGER PRIMARY KEY,
+                            Name TEXT NOT NULL ''' +
+                      ('UNIQUE' if unique_by_fields else '') + ''',
+                            Surname TEXT NOT NULL ''' +
+                      ('UNIQUE' if unique_by_fields else '') + ''',
```

Comment:
code_duplication.
another way to put constrain: https://www.w3schools.com/mysql/mysql_unique.asp